client/client.go

```diff
@@ -454,3 +454,18 @@ func (client *Client) CreateUsers(options []*CreateUserOptions) ([]*CreateUserRe
 	}
 	return results, nil
 }
+
+type DeviceUser struct {
```

Comment:
Also, should this be simply "User"? This is probably the most first-class idea of users we have. Any other users would need such a qualificator (e.g. StoreUser, if we need that some day).